modules/api/docs/LorisRESTAPI_v0.0.4-dev.md

```diff
@@ -416,10 +416,37 @@ The JSON object is of the form:
 }
 ```
 
-A PUT of the same format but with only the Meta fields will create the VisitLabel
+A PUT request with only the Meta fields will create the VisitLabel
 for this candidate, in an unstarted stage if the Visit label provided is valid.
 
-PATCH is not supported for Visit Labels.
+A PATCH request of the form:
+```js
+{
+    "CandID"  : 317604,
+    "Visit"   : 'Visit 01',
+    "Site"    : 'DCC',
+    "Battery" : 'Stale',
+    "Project" : 'Pumpernickel',
+    "Stages" : {
+        "Screening" :  {
+            "Date" : "YYYY-MM-DD",
+            "Status" : "Pass|Failure|Withdrawal|In Progress"
+        },
+        "Visit" : {
+            "Date" : "YYYY-MM-DD",
+            "Status" : "Pass|Failure|Withdrawal|In Progress"
+        },
+        "Approval" : {
+            "Date" : "YYYY-MM-DD",
+            "Status" : "Pass|Failure|Withdrawal|In Progress"
+        }
+    }
+}
+```
+will update the stages date and status. 
+
+At this moment, we only support requests for the Visit stage with the In progress status,
```

Comment:
It's not clear to me what this means for the other fields. Should they be left out of the request? Included with garbage data?

Reply:
I removed the other fields.